### PR TITLE
Nav Item

### PR DESCRIPTION
**Pull request by joshwooding**

### CODE REVIEW

**Comment by james-nash** [comment]
Nice work!

I like the API approach you've taken!

File: packages/lab/src/nav-item/NavItem.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  background: var(--salt-navigable-primary-background);
+  gap: var(--salt-spacing-75);
+}
+
+.saltNavItem-horizontal {
+  min-height: calc(var(--salt-size-stackable) + var(--salt-size-accent) + var(--salt-spacing-100));
```
Comment (libertymayc) [inline packages/lab/src/nav-item/NavItem.css:11]:
```suggestion
  min-height: calc(var(--salt-size-stackable) + var(--salt-size-bar) + var(--salt-spacing-100));
```

Comment (libertymayc) [reply packages/lab/src/nav-item/NavItem.css:11]: new token update + stackable will be deprecated, i think it should be size-base and then padding above and below of spacing-50, or whatever this is calculated to be

Comment (joshwooding) [reply packages/lab/src/nav-item/NavItem.css:11]: Not sure if it's worth tackling this atm, versus when we have design guidance

Comment (libertymayc) [reply packages/lab/src/nav-item/NavItem.css:11]: np

**Review by libertymayc — APPROVED** [review]

File: packages/lab/stories/nav-item/nav-item.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+import { Card, H2, Link } from "@salt-ds/core";
+
+import "./nav-item.stories.css";
+
+export default {
+  title: "Lab/Nav Item",
+  component: NavItem,
+};
+
+const Template: Story<NavItemProps> = (args) => <NavItem {...args} />;
```
Comment (alycrys) [inline packages/lab/stories/nav-item/nav-item.stories.tsx:13]: I think Template is not necessary here

**Review by alycrys — APPROVED** [review]

**Pull request merged**